be/src/Project/Services/Identity/Identity.Api/Areas/Home/ViewModels/PrivacyPolicyPageViewModel.cs

```diff
@@ -8,7 +8,7 @@ public class PrivacyPolicyPageViewModel
     {
         public MetadataViewModel Metadata { get; set; }
         public HeaderViewModel Header { get; set; }
-        public ContentPageViewModel Content { get; set; }
+        public PolicyPageViewModel Policy { get; set; }
```

Comment:
**suggestion:** Can you set to PolicyDetails or something similar instead of PolicyPage? You are in the PolicyPageViewModel